core/new-gui/src/app/workspace/component/workspace.component.ts

```diff
@@ -21,15 +21,21 @@ import { StubOperatorMetadataService } from '../service/operator-metadata/stub-o
     WorkflowActionService,
     WorkflowUtilService,
     DragDropService,
-    ExecuteWorkflowService
+    ExecuteWorkflowService,
+    ResultPanelToggleService
   ]
 })
 export class WorkspaceComponent implements OnInit {
 
-  constructor(
-  ) { }
+  public showPanel: boolean | undefined;
```

Comment:
`showPanel` could be initialized to false
`public showPanel = false`

rename `showPanel` to `showResultPanel` globally.